modules/vaos/spec/models/ccra/referral_detail_spec.rb

```diff
@@ -11,175 +11,93 @@
         'Referral' => {
           'ReferralExpirationDate' => '2024-05-27',
           'CategoryOfCare' => 'CARDIOLOGY',
-          'TreatingProvider' => 'Dr. Smith',
           'TreatingFacility' => 'VA Medical Center',
           'ReferralNumber' => 'VA0000005681',
-          'network' => 'Veteran Affairs Payment',
-          'networkCode' => 'VA',
-          'referralConsultId' => '984_646907',
-          'referralDate' => '2024-07-24',
-          'referralLastUpdateDateTime' => '2024-07-25T10:30:00',
-          'referringFacility' => 'Bath VA Medical Center',
-          'referringProvider' => 'ERMIAS YIRGA',
-          'seocId' => 'MSC_CARDIOLOGY_1.4.17_REV_PRCT',
-          'seocKey' => '23',
-          'serviceRequested' => 'Cardiology_REV_PRCT SEOC 1.4.17',
-          'sourceOfReferral' => 'Interfaced from VA',
-          'sta6' => '534',
-          'stationId' => '528A6',
-          'status' => 'Sent',
-          'treatingFacilityFax' => 'na',
-          'treatingFacilityPhone' => '505-248-4062',
-          'appointments' => [
-            {
-              'appointmentCreateDateTime' => '2024-05-01 17:08:17',
-              'appointmentDate' => '2025-05-07',
-              'appointmentFor' => 'Anesthesia consultation',
-              'appointmentStatus' => 'X',
-              'appointmentTime' => '13:00:00'
+          'ReferralDate' => '2024-07-24',
```

Comment:
Note: I had to remove some tests from this file to keep the LOC low. I have added those tests in this PR tbd